LICENSE-binary

```diff
@@ -309,6 +309,7 @@ org.slf4j:jcl-over-slf4j
 org.webjars:swagger-ui
 org.xerial.snappy:snappy-java
 org.yaml:snakeyaml
+info.picocli:picocli
```

Comment:
Please keep this license files sorted in alphabetical order.

Reply:
sure updated